Use SAP help documents for Kyma references in the observability section (#115)
SAP-samples/btp-developer-guide-cap: documentation/observability/5-deploy-to-kyma.md

# Enable Observability in the SAP BTP, Kyma Runtime

## Prerequisites

- Add the Kyma [Telemetry module](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-telemetry-module) by following the steps described in [Add and Delete a Kyma Module Using Kyma Dashboard](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module).
- Have an entitlement for the service `cloud-logging` assigned to your subaccount.

## Set Up SAP Cloud Logging and integrate it with the SAP BTP, Kyma runtime

> To create a SAP Cloud Logging instance and start streaming logs, metrics and traces from SAP BTP, Kyma runtime to the instance, you need to follow these steps:

Reviewer: ```suggestion
To create an SAP Cloud Logging instance and start streaming logs, metrics, and traces from SAP BTP, Kyma runtime to the instance, you need to follow these steps:
```

1. Log in to the Kyma cluster.

   > [!TIP]

>    > To log in to the Kyma cluster, follow the [Access a Kyma Instance Using kubectl](https://help.sap.com/docs/btp/sap-business-technology-platform/access-kyma-instance-using-kubectl) instructions.

Reviewer: ```suggestion
   > To log in to the Kyma cluster, follow [Access a Kyma Instance Using kubectl](https://help.sap.com/docs/btp/sap-business-technology-platform/access-kyma-instance-using-kubectl).
```

To start streaming logs, metrics and traces from SAP BTP, Kyma runtime to the SAP Cloud Logging service, you need to follow these steps:

1. Add the Kyma [Telemetry module](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-telemetry-module) by following the steps described in [Add and Delete a Kyma Module Using Kyma Dashboard](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module).

> 2. An instance of SAP Cloud Logging with OpenTelemetry is enabled to ingest distributed traces and metrics.

Reviewer: This is phrased more like a prerequisite. For a step, I'd suggest using imperative.

Reviewer: Oh wait, that comes in line 17? That seems a bit redundant to me.

Reviewer: Then I'd clearly separate the overview/intro paragraph from the step instructions.
Basically, saying

> To start streaming logs, metrics and traces from SAP BTP, Kyma runtime to the SAP Cloud Logging service, you need the Kyma Telemetry module and an instance of SAP Cloud Logging with OpenTelemetry:

1. Add the Telemetry module. For details, see [Add and Delete](URL).
2. To create the SAP Cloud Logging service instance, first, log in to the Kyma cluster.
   > [! TIP]
   > We recommend...
3. Create a namespace...

etc.

Reviewer: Ignore; this requires bigger changes which should happen in a different PR.

Author: done

> [!TIP]
>We recommend that you create an instance of SAP Cloud Logging with the SAP BTP service operator, because it takes care of creation and rotation of the required secret. See [Create an SAP Cloud Logging Instance through SAP BTP Service Operator](https://help.sap.com/docs/cloud-logging/cloud-logging/create-sap-cloud-logging-instance-through-sap-btp-service-operator)
However, you can choose any other method of creating the instance and the secret, as long as the parameter for the OTLP ingestion is enabled in the instance. For details, see [Configuration Parameters](https://help.sap.com/docs/cloud-logging/cloud-logging/configuration-parameters).

To create the SAP Cloud Logging service instance, follow these steps:

1. Log in to the Kyma cluster.

> [!TIP]
> To log in to the Kyma cluster, follow the [Access a Kyma Instance Using kubectl](https://help.sap.com/docs/btp/sap-business-technology-platform/access-kyma-instance-using-kubectl) instructions.

1. Create a namespace with the following command:
   
    ``` 
    kubectl create namespace sap-cloud-logging-integration 
    ```

> 2. Create a new file called `cloud-logging-instance.yaml` and paste the following code:

Reviewer: Numbering is rendered weirdly here: 1, 2, 1, 1, 2, 3...

<img width="103" alt="image" src="https://github.com/user-attachments/assets/10cb84d6-1a11-4927-b472-f8bec42af904">

Author: fixed

        ingest_otlp:
          enabled: true
    ```

>     The `ingest_otlp` will enable support for OpenTelemetry based data, which is required for the ingestion of distributed traces and metrics.

Reviewer: ```suggestion
    The `ingest_otlp` will enable support for OpenTelemetry-based data, which is required for the ingestion of distributed traces and metrics.
```

1. Apply the configuration with the following command:

    ```
    kubectl apply -n sap-cloud-logging-integration -f cloud-logging-instance.yaml 
    ```
    This will trigger the provisioning of a SAP CLoud Logging instance
   > [!TIP]

>    > In this guide you create an instance of SAP Cloud Logging with the SAP BTP service operator, which is the recommended way. It takes care of creation and periodic rotation of the required secret. See [Create an SAP Cloud Logging Instance through SAP BTP Service Operator](https://help.sap.com/docs/cloud-logging/cloud-logging/create-sap-cloud-logging-instance-through-sap-btp-service-operator)

Reviewer: ```suggestion
   > In this guide, you create an instance of SAP Cloud Logging with the SAP BTP service operator, which is the recommended way. It takes care of creation and periodic rotation of the required secret. See [Create an SAP Cloud Logging Instance through SAP BTP Service Operator](https://help.sap.com/docs/cloud-logging/cloud-logging/create-sap-cloud-logging-instance-through-sap-btp-service-operator).
```

        ingest_otlp:
          enabled: true
    ```
    The `ingest_otlp` will enable support for OpenTelemetry based data, which is required for the ingestion of distributed traces and metrics.

1. Apply the configuration with the following command:

    ```
    kubectl apply -n sap-cloud-logging-integration -f cloud-logging-instance.yaml 
    ```

>     This will trigger the provisioning of a SAP CLoud Logging instance

Reviewer: ```suggestion
    This will trigger the provisioning of a SAP Cloud Logging instance
```

      secretName: sap-cloud-logging
      credentialsRotationPolicy:
        enabled: true
        rotationFrequency: 1440h

    ```
    To apply the configuration, use the following command:
    ```
    kubectl apply -n sap-cloud-logging-integration -f cloud-logging-binding.yaml
    ```
    
1. Ship the logs to the SAP Cloud Logging service. See [Ship Logs to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-logs-to-sap-cloud-logging).

1. Ship the distributed traces to the SAP Cloud Logging service. See [Ship Distributed Traces to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-distributed-traces-to-sap-cloud-logging).

1. Ship the metrics to the SAP Cloud Logging service. See [Ship Metrics to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-metrics-to-sap-cloud-logging).

    Once the configurations are applied, you can check the status of all telemetry configurations in the Kyma dashboard. They should look like this: 

>     <img src="./images/pipelines.png" />

Reviewer: In the screenshot, consider blurring out irrelevant information; especially parts of the navigation and specific info like `kyma-garden-stage--e60c7...`

    > [!TIP]
    > Learn how to access [Kyma dashboard](https://learning.sap.com/learning-journeys/deliver-side-by-side-extensibility-based-on-sap-btp-kyma-runtime/using-the-kyma-dashboard_d23b12a1-d17c-491d-a80b-cb78039e317e).

> 1.  [Kyma Dashboard Integration](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-metrics-to-sap-cloud-logging).

Reviewer: This is just a link, no instruction. Following the pattern of the steps above, maybe something like this: 

```suggestion
1.  For easier access, adjust the Kyma dashboard navigation. See [Kyma Dashboard Integration](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=set-up-kyma-dashboard-integration).
```

> # Deploy the Incident Management Application in the SAP BTP, Kyma Runtime

Author: done

> 

Reviewer: Line 95: Why is this just a lonely link as part of a list with 1 item? Could this be a sentence similar to line 92 usw?

Author: Made a proposal

Reviewer: LGTM

3. Apply the configuration with the following command:

    ```
    kubectl apply -n sap-cloud-logging-integration -f cloud-logging-instance.yaml 
    ```
       

> 4. Wait for the instance to be created, you can check its status using the following command:

Reviewer: ```suggestion
4. Wait for the instance to be created. You can check its status using the following command:
```

Author: fixed

    ```
    To apply the configuration, use the following command:
    ```
    kubectl apply -n sap-cloud-logging-integration -f cloud-logging-binding.yaml
    ```
    

> 7. Ship the logs to the SAP Cloud Logging service. See [Ship Logs to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-logs-to-sap-cloud-logging).

Reviewer: It's rendered correctly, but in the raw version it looks weird to jump from step 5 (line 54) to 7 (line 75).

Author: Please check once more, I changed ordering
